libs/zflecs/src/zflecs.zig

```diff
@@ -2534,6 +2565,8 @@ pub fn ADD_SYSTEM_WITH_FILTERS(
     filters: []const term_t,
 ) void {
     var desc = SYSTEM_DESC_WITH_FILTERS(fn_system, filters);
+    // std.json.stringify(desc, .{}, std.io.getStdOut().writer()) catch unreachable;
+    std.debug.print("lol234 {any}", .{desc});
```

Comment:
Can we remove these lines?

Reply:
Oops!

Reply:
Fixed! Thanks for noticing! @hazeycode can we do a squash and erase my failboating? :)

Reply:
Of course ;)